execute/table/chunk.go

```diff
@@ -110,7 +110,7 @@ func (v Chunk) Ints(j int) *array.Int {
 
 // Uints is a convenience function for retrieving an array
 // as a uint array.
-func (v Chunk) Uints(j int) *array.Uint {
+func (v Chunk) UInts(j int) *array.Uint {
```

Comment:
This was intentionally named this way to match with arrow's naming convention.

Reply:
I modified this, and added the `Times()` method, so that `table.Chunk` could implement the `flux.ColReader()` interface. That way we can pass it directly to `EvalRow()`. 

If there's a better way to do that though, I'm happy to undo this change.

Reply:
The better way to do it is to do this:

```go
buf := chunk.Buffer()
EvalRow(&buf, ...)
```

That method returns `arrow.TableBuffer` which implements `flux.ColReader`. It's there to facilitate the older code that still uses `flux.ColReader`. I'd prefer to avoid adding methods to `table.Chunk` to match `flux.ColReader` as newer code should use the struct directly rather than the interface.

Reply:
Makes sense. I'll change it.